datastream-server/src/main/java/com/linkedin/datastream/server/zk/KeyBuilder.java

```diff
@@ -60,10 +60,16 @@ private KeyBuilder() {
    */
   private static final String DATASTREAM_TASK_CONFIG = CONNECTOR + "/%s/config";
 
+
+  /**
+   * Task lock root path under connectorType
+   */
+  private static final String DATASTREAM_TASK_LOCK_ROOT = CONNECTOR + "/lock";
```

Comment:
The existing connector have lock under task. i.e. task/lock. Since the task will get deleted when the assignment is removed, the lock for existing connector will get cleaned up. As a result, we don't need to do a clean up job. That's also the reason to make this change. The lock shouldn't be cleaned up right after the task is removed from coordinator assignment.